pkg/review/commit_review_status_test.go

```diff
@@ -0,0 +1,443 @@
+                     "startCursor": ""
+                   },
+                   "totalCount": 2
+                 }
+               }
+             }
+           }
+         }`,
+				`{
+           "data": {
+             "repository": {
+               "object": {
+                 "associatedPullRequests": {
+                   "nodes": [
+                     {
+                       "databaseId": 2,
+                       "number": 48,
+                       "reviewDecision": "REVIEW_REQUESTED"
+                     }
+                   ],
+                   "pageInfo": {
+                     "endCursor": "FG",
+                     "hasNextPage": false,
+                     "hasPreviousPage": true,
+                     "startCursor": "XQ"
+                   },
+                   "totalCount": 2
+                 }
+               }
+             }
+           }
+         }`,
+			},
+		},
+		{
+			name:       "No Associated Pull Requests for a commit",
+			token:      "ghp_JQqAJky0GlNB7xqVCbUrivIgFZ09V8gGCi5C",
+			githubOrg:  "test-org",
+			repository: "test-repo",
+			commitSha:  "kof6p96lr6qvdu81qw49fhmoxrod9qmc2qak51nh",
+			wantRequestBodies: []string{
+				`{
+           "query": "
+             query($commitSha:GitObjectID! $githubOrg:String! $pageCursor:String! $repository:String!) {
+               repository(owner: $githubOrg, name: $repository) {
+                 object(oid: $commitSha) {
+                   ... on Commit{
+                     associatedPullRequests(first: 100, after: $pageCursor) {
+                       nodes{
+                         databaseId,
+                         number,
+                         reviewDecision
+                       },
+                       pageInfo{
+                         endCursor,
+                         hasNextPage,
+                         hasPreviousPage,
+                         startCursor
+                       },
+                       totalCount
+                     }
+                   }
+                 }
+               }
+             }
+           ",
+           "variables": {
+             "commitSha": "kof6p96lr6qvdu81qw49fhmoxrod9qmc2qak51nh",
+             "githubOrg": "test-org",
+             "pageCursor": "",
+             "repository":"test-repo"
+           }
+         }`,
+			},
+			want: []*PullRequest{},
+			responseBodies: []string{
+				`{
+           "data": {
+             "repository": {
+               "object": {
+                 "associatedPullRequests": {
+                   "nodes": [],
+                   "pageInfo": {
+                     "endCursor": "XQ",
+                     "hasNextPage": false,
+                     "hasPreviousPage": false,
+                     "startCursor": ""
+                   },
+                   "totalCount": 0
+                 }
+               }
+             }
+           }
+         }`,
+			},
+		},
+	}
+	for _, tc := range cases {
```

Comment:
Should run cases as sub-tests `t.Run(tc.name, func(t *testing.T) { ... })` https://github.com/abcxyz/readability/wiki/Go-Best-Practices#parallelizable

Reply:
done.